assets/js/components/ClusterDetails/ClusterSettings.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+
+import { EOS_ARROW_BACK } from 'eos-icons-react';
+import Button from '@components/Button';
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ');
+}
+
+import { Tab } from '@headlessui/react';
+import { ChecksSelection } from './ChecksSelection';
+
+export const ClusterSettings = () => {
+  const { clusterID } = useParams();
+  const navigate = useNavigate();
+
+  const clusters = useSelector((state) => state.clustersList.clusters);
+  const cluster = clusters.find((cluster) => cluster.id === clusterID);
```

Comment:
Could we use the `getCluster` selector function here?
https://github.com/trento-project/web/blob/main/assets/js/state/selectors/index.js#L7

Reply:
definitely!